src/libs/defiPositions/types.ts

```diff
@@ -0,0 +1,25 @@
+export enum AssetType {
+  Liquidity,
+  Collateral,
+  Borrow
+}
+
+export interface PositionAsset {
+  address: string
+  symbol: string
+  decimals: number
+  amount: number
+  type: AssetType
+  additionalData: any | undefined
+}
+
+export interface Position {
+  providerName: string
+  positionType: string
+  assets: PositionAsset[]
+  network: string
```

Comment:
rename to networkId

Reply:
Wait, it's actually the name of the network. Why is it not the networkId?
![image](https://github.com/user-attachments/assets/b74fd024-71d9-4f08-b0e4-a68e90909905)